src/neo4j_graphrag/embeddings/openai.py

```diff
@@ -25,26 +26,29 @@
     openai = None  # type: ignore
 
 
-class OpenAIEmbeddings(Embedder):
+class BaseOpenAIEmbeddings(Embedder, abc.ABC):
```

Comment:
Why have this as a separate base class? Is this too much indirection?

Reply:
What would you suggest instead?